metadata/report/nacos/report.go

```diff
@@ -91,10 +101,10 @@ func (n *nacosMetadataReport) PublishAppMetadata(metadataIdentifier *identifier.
 	if err != nil {
 		return err
 	}
-
+	// same with java impl
 	return n.storeMetadata(vo.ConfigParam{
-		DataId:  metadataIdentifier.GetIdentifierKey(),
-		Group:   n.group,
+		DataId:  metadataIdentifier.Application,
+		Group:   metadataIdentifier.Revision,
 		Content: string(data),
 	})
```

Comment:
We need to adapt the previous rules here

Reply:
Otherwise, the previous service will not be able to call the current service

Reply:
OK，I will fix it